### PR TITLE
Bugfix/manager ids disappearing service restart

### LINKED ISSUE #1164: Published API backends are not rendered in 'Manage API Backend' view
Findings: 
- Previously added API backends are not rendered in 'Manage API Backend view':shows a blank page. 
- Shows only APIs added on current day in 'Manage API backend' view and they get disappeared later. 

Screenshot: 
selection_001

### PR DESCRIPTION
**Pull request by jykae**
Closes #1164 

## Proposed changes
- handle addition of API backend separately for Umbrella & Apinf, Apinf has additional fields we need, like managerIds.
- move database permissions under server (because allow/deny are server only)
- use defaultValue instead of autoValue for managerIds
- add imports

### CODE REVIEW

**Comment by jykae** [comment]
Last commit is for keeping the functionality the same way as it is with synced API backends that come from Umbrella without owner.

**Comment by jykae** [comment]
@apinf/developers please review

**Comment by frenchbread** [comment]
Good job @jykae!

File: apis/client/api_backends/add/wizard.js [hunk 1]
```diff
@@ -96,11 +98,6 @@ Template.addApiBackendWizard.helpers({
         // Add the least connections balance algorithm setting
         apiBackend.balance_algorithm = "least_conn";
 
-        // Delete unneeded properties: database insert/update related
-        delete apiBackend.insertDoc;
```
Comment (brylie) [inline apis/client/api_backends/add/wizard.js]: @frenchbread please make sure these `apiBackend` properties do not get stored in our collection:

- `insertDoc`
- `updateDoc`

File: apis/client/api_backends/add/wizard.js [hunk 2]
```diff
@@ -109,13 +106,17 @@ Template.addApiBackendWizard.helpers({
           Meteor.call('createApiBackendOnApiUmbrella', apiBackend, function(error, apiUmbrellaWebResponse) {
             if (apiUmbrellaWebResponse.http_status === 200) {
               // Get API Backend from API Umbrella response
-              var apiBackend = apiUmbrellaWebResponse.result.data.api;
+              var apiBackendCreated = apiUmbrellaWebResponse.result.data.api;
 
+              // Create object for insert to Apinf
+              let newApiBackend = apiBackendCreated;
```
Comment (brylie) [inline apis/client/api_backends/add/wizard.js]: What is the purpose of this step? Why not use the `apiBackendCreated` object directly in the `ApiBackends.insert()`?

File: apis/collection/backend.js [hunk 3]
```diff
@@ -650,15 +650,7 @@ Schemas.ApiBackendsSchema = new SimpleSchema({
   managerIds: {
     type: [String],
     regEx: SimpleSchema.RegEx.Id,
-    autoValue: function () {
```
Comment (brylie) [inline apis/collection/backend.js]: How does removing the `autoValue' fix this issue? What steps can we take to reproduce the problem and verify this fixes things?

**Pull request merged**